tools/check-changelog-entry-file/main.go

```diff
@@ -46,7 +46,7 @@ func main() {
 		return
 	}
 
-	log.Fatalf("Consider using label %s if this PR doesn't need a changelog entry file. Read contributing guides for more info.\nChangelog file not found: %s, err: %v", skipLabelName, filePath, errFile)
+	log.Fatalf("Changelog file %s not found due to the following reason: %v.\nHave you ran the `make generate-changelog-entry` command?\nIf this PR doesn't need a changelog entry file, consider using label %s.\nRead contributing guides (https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/contributing/changelog-process.md) for more info.", filePath, errFile, skipLabelName)
```

Comment:
@lantoli do you prefer this more? Feel free to suggest a proposal as well: end goal is to make sure the author quickly sees the command to run once they run into a failure of this check.

```suggestion
	log.Fatalf("Have you ran the `make generate-changelog-entry` command?\nIf this PR doesn't need a changelog entry file, consider using label %s.\nRead contributing guides (https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/contributing/changelog-process.md) for more info.\nChangelog file %s not found due to the following reason: %v.", skipLabelName, filePath, errFile)
```

Reply:
yeah, that's good got me, knitpit:   Changelog file -> Changelog entry file. (agustin told me this in multiple comments with good reason :-) )